andela/trime
# [Chore #117876129] Can now run e2e tests with gulp

#### What does this PR do?

Adds the ability to run e2e tests with gulp.

#### Description of Task to be completed?

Migration of all tasks from npm to gulp, this PR completes the migration.

#### How should this be manually tested?

After cloning the repo and installing node modules run `gulp e2etest` from the project folder in the terminal (ensure Google Chrome is installed).

All e2e tests should run.

#### What are the relevant pivotal tracker stories?

[Chore #117876129] (https://www.pivotaltracker.com/projects/1454736/stories/117876129)
